data/json/items/gun/shot.json

```diff
@@ -606,7 +606,7 @@
     "variants": [
       {
         "id": "streetsweeper",
-        "name": { "str": "Cobray Streetsweeper" },
+        "name": { "str": "Cobray Streetsweeper revolver shotgun" },
```

Comment:
Could just leave it called a shotgun, to be honist. The fact that it's a revolving shotgun makes no difference in game.